stable/admin/values.yaml

```diff
@@ -80,6 +80,20 @@ admin:
       - ReadWriteOnce
     # storageClass: "-"
 
+  ## Enable persistent log volumes to retain logs when an external logging solution is not used.
+  logPersistence:
+    enabled: false
+    overwriteBackoff:
+      # Copies of the crash directory are taken to avoid overwrites of root crash.
+      # This controls the window within which no more copies of the crash directory will be taken to avoid the disk filling.
+      # Default will retain 3 copies within the last 120 minutes, after which they will resume.
```

Comment:
I don't understand what "they will resume" is intended to mean - but I assume "after which the directories will be overwritten and/or cleaned up"?

Reply:
No this won't ever overwrite or clear up the backups, since we don't want to risk losing the root dump.  It will just slow down disk filling up with a rapid crashloop.  This expects disk space is managed for persistent volumes in the same way it should be for archives and backup volumes...

Reply:
updated for clarity